src/User_Command.php

```diff
@@ -1197,4 +1197,41 @@ private function update_msuser_status( $user_ids, $pref, $value ) {
 		Utils\report_batch_operation_results( 'user', $verb, count( $user_ids ), $successes, $errors );
 	}
 
+	/**
+	 * Checks if given user credentials are valid or not.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <user>
+	 * : The user login or user email of the user to check credentials.
```

Comment:
Should mention ID as well so `The user login, user email or user ID of the user to check credentials for.`